patches/545408A7 - Grand Theft Auto V (TU18).toml

```diff
@@ -0,0 +1,16 @@
+title_name = "Grand Theft Auto V"
+title_id = "545408A7" # TT-2215
+#hash = "" # default.xex
```

Comment:
🤔

Reply:
Waiting for gliniak new hash gen to be merged, otherwise I'll have to change it again